ground/src/main/java/com/google/android/ground/model/mutation/PhotoMutation.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.model.mutation
+
+/** Represents a modification of a photo in remote storage. */
+data class PhotoMutation(val submissionMutationId: String, val remotePath: String) : Mutation()
```

Comment:
If Photo is just a special Value class, wouldn't SubmissionMutation already cover this? Or is that too coarse grained to capture the photo mutation? I think the easiest way of doing this with the current model would be to make SubmissionMutation more fine-grained (per value rather than per submission).

Reply:
Yep, that approach sounds way better. We can just handle this as part of the submission mutation.